test/micro/org/openjdk/bench/jdk/incubator/foreign/LoopOverNonConstantHeap.java

```diff
@@ -97,11 +97,6 @@ public void setup() {
         byteBuffer = ByteBuffer.wrap(base).order(ByteOrder.nativeOrder());
     }
 
-    @TearDown
```

Comment:
This change fix an issue with the benchmark which was caused by the new ResourceScope API: heap segments are associated with the global scope, so they cannot be closed.